imports/startup/server/index.coffee

```diff
@@ -168,6 +168,43 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date, date2, days) ->
+    query = prefixes + """
+      SELECT ?resolvedTerm
+        (sample(?termLabel) as ?word)
+		    (sample(?termLabel2) as ?word2)
+        (count(DISTINCT ?article) as ?count)
+		    (sample(?c2) as ?count2)
+        ((?count2)/(xsd:float(#{escape(days)}*4)) as ?rate2)
+        ((?count)/(xsd:float(#{escape(days)})) as ?rate)
+  		  ((xsd:float(?rate)/(xsd:float(?rate2))) AS ?result)
+      WHERE {
+        ?phrase anno:category "diseases"
+        ; ^dc:relation ?resolvedTerm
+        ; anno:source_doc ?article
+        .
+        ?article pro:post/pro:date ?dateTime .
+        ?resolvedTerm rdfs:label ?termLabel
+        FILTER (?dateTime > "#{escape(date)}"^^xsd:dateTime)
+
+        {SELECT (count(distinct ?article2) as ?c2) ?resolvedTerm ?termLabel2
+    	     WHERE{
+            ?prev_mention anno:source_doc ?article2
+            ; ^dc:relation ?resolvedTerm
+            .
+            ?article2 pro:post/pro:date ?dateTime2 .
```

Comment:
We should only use the post date if there is no article date. See the query here for an example of how to default to the article date when the post date doesn't exist: https://github.com/ecohealthalliance/nia-monitor/blob/4c4ee7d56dd000280307932b2f8d7af98f579a5a/imports/startup/server/index.coffee#L78